src/components/molecules/Visualizer/Infobox/index.tsx

```diff
@@ -82,9 +83,16 @@ const Infobox: React.FC<Props> = ({
       sceneProperty={sceneProperty}
       title={property?.default?.title || title}
```

Comment:
You can assign default values here like `title` if needed

Reply:
didin't get it ... no changes are applied from my side her